lib/workerholic/job_wrapper.rb

```diff
@@ -4,16 +4,16 @@ class JobWrapper
     attr_reader :klass, :arguments, :statistics
 
     def initialize(options={})
-      @klass = options[:class]
+      @klass = options[:klass]
       @arguments = options[:arguments]
       @execute_at = options[:execute_at]
       @retry_count = options[:retry_count] || 0
-      @statistics = Statistics.new(options[:statistics] || {})
+      @statistics = JobStatistics.new(options[:statistics] || {})
     end
 
     def to_hash
       {
-        class: klass,
+        klass: klass,
```

Comment:
same as before, why rename it to klass when class was working fine? This `class` was only used in the context of a key. That's what `ActiveJob` does by default, as long as it's a key. I think we should keep 'class'.